### PR TITLE
Fixed russian translation

### PR DESCRIPTION
**Pull request by kerastinell**
I've corrected typos @LvMsterfild made in Russian translation (PR #1427)
This PR is replacement of broken #1440

### CODE REVIEW

File: translations/monero-core_ru.ts [hunk 1]
```diff
@@ -1058,12 +1058,12 @@
...
         <location filename="../pages/Settings.qml" line="507"/>
         <source>Custom decorations</source>
-        <translation>Переключить режим окна</translation>
+        <translation>Несистемное обрамление окна</translation>
     </message>
     <message>
         <location filename="../pages/Settings.qml" line="517"/>
         <source>Log level</source>
-        <translation>Уровень логирования</translation>
+        <translation>Уровень логгирования</translation>
```
Comment (troggy) [inline translations/monero-core_ru.ts]: I would leave the existing translation here. Never saw "логгирование" in the wild

**Comment by luigi1111** [comment]
@kerastinell please rebase.

**Comment by kerastinell** [comment]
I was trying to rebase using different tools for hours, but this page still shows 2 unresolved conflicts. Not cool, looks like I seriously f-d up somewhere. Will close this PR today if I couldn't find a solution to this.

**Comment by erciccione** [comment]
hey @kerastinell you solved the conflicts, but now you have a merge commit. We usually avoid those. I can fix this PR for you, just add me as a collaborator to your monero-gui repo

**Pull request merged**